includes/admin/views/html-admin-setup-step-1.php

```diff
@@ -23,7 +23,7 @@
 	<div class="crowdsignal-setup__content">
 		<div class="crowdsignal-setup__description">
 				<h1><?php esc_html_e( 'Welcome to Crowdsignal Forms', 'crowdsignal-forms' ); ?></h1>
-				<p><?php echo wp_kses_post( 'To collect and manage responses you need to connect the plugin to <a href="https://crowdsignal.com">Crowdsignal</a>. <br />It will take less than a minute and it’s free.', 'crowdsignal-forms' ); ?></p>
+				<p><?php echo wp_kses_post( __( 'To collect and manage responses you need to connect the plugin to <a href="https://crowdsignal.com">Crowdsignal</a>. <br />It will take less than a minute and it’s free.', 'crowdsignal-forms' ) ); ?></p>
```

Comment:
We generally don't like the URLs to be part of the translation, to avoid any potential issues with translators breaking the link.

Would you mind following the pattern used here? https://github.com/Automattic/crowdsignal-forms/blob/e4c2eb37e4898cf68eb0908483f295be647ffafc/includes/admin/views/html-admin-dashboard-teaser.php#L41-L48

Reply:
Hi @jgcaruso 

I made the changes.

Thanks.